Tasks/AzureSpringCloudV0/operations/taskparameters.ts

```diff
@@ -2,7 +2,7 @@ import { Package, PackageType } from 'azure-pipelines-tasks-webdeployment-common
 
 export class Inputs {
     public static readonly connectedServiceName = 'ConnectedServiceName';
-    public static readonly azureSpringCloud = 'AzureSpringCloud';
+    public static readonly azureSpringCloud = 'AzureSpringApps';
```

Comment:
If won't introduce breaking, how about change this param name "azureSpringCloud" as well?

Reply:
Yes, this could introduce breaking, I will change it back.